lib/run-in-terminal.coffee

```diff
@@ -356,68 +370,68 @@ module.exports =
 
             start_path = path.dirname(start_path) if not should_run
             if request_arguments
-
-                @arguments_view.show(start_path)
-
+              @arguments_view.show(start_path)
             else
-
-                start_terminal(start_path)
+              start_terminal(start_path)
```

Comment:
Hmm, not sure what happened there, probably testing something. ;) Reversed it to the original.